sources/Inputs/src/NetworkManager.cpp

```diff
@@ -290,7 +290,18 @@ NetworkManager::buildTree() {
       isConverter1Rectifier = true;
     }
     auto hvdcLineCreated =
-        HvdcLine::build(hvdcLine->getID(), converterType, converter1, converter2, activePowerControl, hvdcLine->getPmax(), isConverter1Rectifier);
+        HvdcLine::build(hvdcLine->getID(),
+                        converterType,
+                        converter1,
+                        converter2,
+                        activePowerControl,
+                        hvdcLine->getPmax(),
+                        isConverter1Rectifier,
+                        hvdcLine->getVNom(),
+                        hvdcLine->getActivePowerSetpoint(),
+                        hvdcLine->getResistanceDC(),
+                        hvdcLine->getConverter1()->getLossFactor() / 100.,
```

Comment:
Please use converterDyn1 (resp. converterDyn2) instead of hvdcLine->getConverter1() (resp. hvdcLine->getConverter2())